hw_4/script.js

```diff
@@ -0,0 +1,25 @@
+function factorial(n) {
+  return n <= 1 ? 1 : n * factorial(n - 1);
+}
+
+var arr1 = [];
+
+function heapFilling() {
+  for (let i = 0; i < 1024 * 1024 * 108; i++) {
```

Comment:
`1024 * 1024 * 108` - It would be good to see a comment on why this particular number is used in this way.

Reply:
https://github.com/BlackTruth/javascript/blob/4d02cb8e9b7adf484665c236844a5d36591978a6/hw_4/script.js#L7-L9

Added comment for a number of loop iterations.